microsite/data/plugins/aws-app-development.yaml

```
@@ -0,0 +1,10 @@
+---
+title: App Development for Backstage.io on AWS
```

May I suggest this title instead as we refer to our website as backstage.io and it might confuse users :) 

```suggestion
title: AWS App Developer Tools
```